samples/nrf_desktop/src/modules/usb_state.c

```diff
@@ -259,7 +259,7 @@ static void device_status(enum usb_dc_status_code cb_status, const u8_t *param)
 
 	case USB_DC_RESUME:
 		__ASSERT_NO_MSG(state == USB_STATE_SUSPENDED);
-		new_state = USB_STATE_POWERED;
+		new_state = USB_STATE_IDLE;
```

Comment:
btw... after resume will we received configured?

Reply:
Not sure how to check it. If I connect device to Windows host and put the host in sleep mode, I get SUSPENDED and DISCONNECTED. After wake up, host sends CONNECTED and all events the same way as on first connect.

Reply:
...even if I disallow computer to turn off USB ports and/or disallow selective USB suspend.